packages/functional-tests/tests/oauth/oauthPermissions.spec.ts

```diff
@@ -58,20 +58,25 @@ test.describe('oauth permissions for trusted reliers', () => {
 
     //Verify sign up code header
     await login.waitForSignUpCodeHeader();
+  });
+});
 
+test.describe('oauth permissions for trusted reliers - sign in ', () => {
+  test.beforeEach(async ({ pages: { login } }) => {
+    test.slow();
+    await login.clearCache();
   });
 
   test('signin without `prompt=consent`', async ({
-    target,
+    credentials,
     pages: { login, relier },
   }) => {
-    await target.auth.signUp(email, password, {
-      lang: 'en',
-      preVerified: 'true',
-    });
     await relier.goto();
     await relier.clickEmailFirst();
-    await login.fillOutEmailFirstSignIn(email, password);
+    await login.fillOutEmailFirstSignIn(
+      credentials.email,
+      credentials.password
+    );
```

Comment:
I'm not sure it's any better, but you could also destructure the `email` and `password` which might make the diffs a bit shorter.

```js
const { email, password } = credentials; // around line 73.5.
```

then you don't need to refactor/rename all these below:
```diff
-email
+credentials.email
```

(but you'd need to do the destructure bit for each test, which might be tedious and a terrible suggestion if the `email`/`password` is only used once per test anyways.)

Reply:
I think to keep all the tests consistent we will have to then destructure it on all the other tests which is a bit of a work for now. We can revisit this idea in the futute.